modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/checkpoint/CheckpointHistory.java

```diff
@@ -90,6 +90,9 @@
     /** Maximum size of WAL archive (in bytes) */
```

Comment:
Lots of reason string constants in this class became unused, we can safely remove them.

Reply:
Removed all *_REASON constants.